tap_twitter_ads/streams.py

```diff
@@ -403,6 +407,59 @@ def sync_endpoint(self,
                             parent_id_limit = child_endpoint_config.parent_ids_limit
                         else:
                             parent_id_limit = 1
+
+                        # Bookmark for child stream
+                        child_last_datetime = self.get_bookmark(state, child_stream_name, start_date)
+                        if not child_last_datetime or child_last_datetime is None:
+                            child_last_datetime = start_date
+                        child_last_dttm = strptime_to_utc(child_last_datetime)
+
+                        child_max_bookmark_value = None
+                        child_counter = 0
+                        # Loop thru cursor records, break out if no more data or child_bookmark_value < child_last_dttm
+                        for record in cursor_child:
+                            # Get dictionary for record
+                            record_dict = self.obj_to_dict(record)
+                            if not record_dict:
+                                # Finish looping
+                                LOGGER.info('Stream: {} - Finished Looping, no more data'.format(stream_name))
+                                break
```

Comment:
Do we need this? The `for` loop will execute till `cursor_child` have data so I think no need to break the loop specifically.

Reply:
No, I have verified that removing `break` will not cause any issue. So, removed it.